syntax = "proto3";

package user_service;

service UserService {
    rpc UpdateBalance(UpdateBalanceRequest) returns (UpdateBalanceResponse);
}

message UpdateBalanceRequest {
    int32 user_id = 1;
    int32 credits = 2;
}

message UpdateBalanceResponse {
    bool success = 1;
}